core/services/mongodb-remote/src/main/java/com/mongodb/stitch/core/services/mongodb/remote/sync/internal/DataSynchronizer.java

```diff
@@ -1839,14 +1839,7 @@ long count(final MongoNamespace namespace, final Bson filter, final CountOptions
 
     try {
       ongoingOperationsGroup.enter();
-      final Lock lock =
-          this.syncConfig.getNamespaceConfig(namespace).getLock().writeLock();
-      lock.lock();
-      try {
-        return getLocalCollection(namespace).countDocuments(filter, options);
```

Comment:
[major] This deleted count

Reply:
no it didn't - look at both sides of the diff

Reply:
Yup whoops